hyperspy/io_plugins/hierarchical.py

```diff
@@ -0,0 +1,665 @@
+from distutils.version import LooseVersion
+import warnings
+import logging
+import datetime
+import ast
+
+import h5py
+
+import numpy as np
+import dask.array as da
+from traits.api import Undefined
+from hyperspy.misc.utils import ensure_unicode, multiply, get_object_package_info
+from hyperspy.axes import AxesManager
+
+version = "3.1"
```

Comment:
Rename to `hspy_reader_version` and import its value from `hyperspy.io_plugins.hspy` to avoid being out of sync?